packages/core/src/stories/announcements/announce-tegel.stories.ts

```diff
@@ -42,7 +42,7 @@ export const Tegel = {
             <article class="tds-u-p2 tds-body-01">
             <section>
                         <p class="tds-body-01"><strong>Published: 2022-04-25</strong></p>
-                        <h1 class="tds-headline-02">Annoucing @scania</h1>
+                        <h1 class="tds-headline-02">Annoucing @scania/tegel</h1>
```

Comment:
@theJohnnyMe any reason this was removed?

Reply:
Must be accident, see no reason not to have it. Great catch!